make/autoconf/flags.m4

```diff
@@ -218,8 +224,8 @@ AC_DEFUN([FLAGS_SETUP_SYSROOT_FLAGS],
     # We also need -iframework<path>/System/Library/Frameworks
     $1SYSROOT_CFLAGS="[$]$1SYSROOT_CFLAGS -iframework [$]$1SYSROOT/System/Library/Frameworks"
     $1SYSROOT_LDFLAGS="[$]$1SYSROOT_LDFLAGS -iframework [$]$1SYSROOT/System/Library/Frameworks"
-    if test -d "[$]$1SYSROOT/System/Library/Frameworks/JavaVM.framework/Frameworks" ; then
-      # These always need to be set on macOS 10.X, or we can't find the frameworks embedded in JavaVM.framework
```

Comment:
I explicitly set this note of macOS 10.X in patch for JDK-8250876. because subframeworks is only available on 10.x, not on 11.x.
Why you want to remove this note ?